src/components/Form.js

```diff
@@ -60,16 +64,16 @@ export default class Form extends Component {
   }
 
   onChange = (formData, options={validate: false}) => {
-    const liveValidate = this.props.liveValidate || options.validate;
-    const errors = liveValidate ? this.validate(formData) :
-                                  this.state.errors;
-    const errorSchema = toErrorSchema(errors);
-    setState(this, {
-      status: "editing",
-      formData,
-      errors,
-      errorSchema
-    }, () => {
+    const mustValidate = this.props.liveValidate || options.validate;
+    let state;
+    if (!mustValidate) {
+      state = {status: "editing", formData};
+    } else {
+      const errors = this.validate(formData);
+      const errorSchema = toErrorSchema(errors);
+      state = {status: "editing", formData, errors, errorSchema};
+    }
+    setState(this, state, () => {
```

Comment:
No, nothing is wrong per se, I'm just trying to understand the changes you made and how they addressed the issue, so that I can better understand the proect and the context for #163 (which changes some of this code again). Anyhow, I'll look at #163 now..